Node/samples/app.js

```diff
@@ -101,6 +104,7 @@ bot.dialog('MentionUser', function (session) {
     var msg = new teams.TeamsMessage(session).text(teams.TeamsMessage.getTenantId(session.message));
     var mentionedMsg = msg.addMentionToText(toMention);
     session.send(mentionedMsg);
+    session.endDialog();
```

Comment:
@BillZengMSFT Without this it can't return to main menu after it's done for `MentionUser` for option 2.